app/components/LeftPaneEntries/TableAndVisualItem/index.tsx

```diff
@@ -0,0 +1,257 @@
+                            value={data?.type}
+                            entryType={entryTypeFromLead}
+                            image={undefined}
+                            entryAttachment={undefined}
+                            imageRaw={data?.filePreview?.url ?? ''}
+                            readOnly
+                        />
+                    )}
+                </Container>
+            </div>
+        );
+    }
+
+    return (
+        <div>
+            <Container
+                className={_cs(
+                    styles.taggedExcerpt,
+                    className,
+                    entryId && isActive && styles.active,
+                )}
+                headerActions={data?.type === 'XLSX' ? (
+                    <QuickActionLink
+                        title="Open external"
+                        to={data?.file?.url || ''}
+                    >
+                        <BsDownload />
+                    </QuickActionLink>
+                ) : undefined}
+                heading={isNotDefined(attachmentId) ? (
+                    <NumberOutput
+                        className={styles.entryId}
+                        prefix="#"
+                        value={Number(attachmentId)}
+                    />
+                ) : (
+                    <span className={styles.unsavedEntry}>(unsaved entry)</span>
+                )}
+                headingClassName={styles.heading}
+                headingSize="extraSmall"
+                headingSectionClassName={styles.headingSection}
+                footerIcons={(
+                    <>
+                        {stale && !deleted && !errored && (
+                            <Tag
+                                spacing="compact"
+                            >
+                                Changed
+                            </Tag>
+                        )}
+                        {deleted && (
+                            <Tag
+                                variant="complement2"
+                                spacing="compact"
+                            >
+                                Deleted
+                            </Tag>
+                        )}
+                        {errored && (
+                            <Tag
+                                spacing="compact"
+                            >
+                                Error
+                            </Tag>
+                        )}
+                    </>
+                )}
+                footerQuickActions={isActive && (
+                    <>
+                        <QuickActionButton
+                            title="Discard changes"
+                            name={attachmentId}
+                            onClick={onDiscardButtonClick}
+                            disabled={disableDiscardButton}
+                        >
+                            <IoClose />
+                        </QuickActionButton>
```

Comment:
I think we need to keep aru actions haru pani, similar to aru entryItems like delete and stuff.